style.css

```diff
@@ -0,0 +1,695 @@
+  text-align: center;
+  justify-items: center;
+  margin: 0 auto;
+  height: 15rem;
+}
+
+.pgrm p {
+  line-height: 20px;
+  margin-top: -20px;
+}
+
+.pgrm h2 {
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: 500;
+  padding: 10px;
+}
+
+.pgrm-footer .pgrm_btn {
+  width: 20.5rem;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  font-size: 20px;
+  border-bottom: solid 1px aliceblue;
+  padding-bottom: 1px;
+}
+
+.pgrm-footer .pgrm_btn2 a {
+  display: block;
+  padding-bottom: 20px;
+  font-size: 18px;
+  width: 21rem;
+}
+
+.pgrm-footer .pgrm_btn a:hover {
+  color: rgb(182, 180, 180);
+}
+
+.pgrm-footer .pgrm_btn2 {
+  display: block;
+  width: 20.5rem;
+  justify-content: center;
+  margin: 0 auto;
+  font-size: 20px;
+  padding: 20px 15px;
+  padding-bottom: 1px;
+  margin-top: 4rem;
+}
+
+.section-ft {
+  background-color: #f3f3f3;
+}
+
+.main-ft {
+  width: 16.5%;
+  justify-items: center;
+  justify-content: center;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.ft-title {
+  font-size: 25px;
+}
+
+.ft-border {
+  display: block;
+  width: 50px;
+  justify-content: center;
+  margin: 0 auto;
+  border-bottom: solid 1px #ec5242;
+}
+
+.ft-speakers {
+  display: flex;
+  gap: 15px;
+}
+
+.ft-speakers-hdn {
+  display: flex;
+  gap: 15px;
+}
+
+.ft-speaker {
+  display: flex;
+  flex-direction: column;
+  width: 24rem;
+  line-height: 15px;
+  margin-bottom: 50px;
+}
+
+.ft-prgph1 {
+  font-style: italic;
+  color: red;
+}
+
+.ft-speaker p,
+h3,
+span {
+  font-size: 12px;
+  margin: 0;
+  margin-bottom: 5px;
+}
+
+.line {
+  border: solid rgb(49, 48, 48) 1px;
+  display: block;
+  width: 20px;
+}
+
+.ft-images {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 15px;
+  justify-items: center;
+  margin: 0 auto;
+  margin-top: 60px;
+  width: 65%;
+}
+
+.more {
+  display: none;
+}
+
+#less-btn {
+  display: none;
+}
+
+footer .foot-sctn1 {
+  background-color: #272a31;
+}
+
+.foot-title {
+  width: 70px;
+  justify-content: center;
+  margin: 0 auto;
+  padding-top: 20px;
+}
+
+.foot-ptnr {
+  font-weight: 500;
+  color: #5d5f63;
+}
+
+.foot-border {
+  border-bottom: #ec5242 solid 2px;
+  display: block;
+  justify-items: center;
+  margin: 0 auto;
+  width: 50px;
+}
+
+.net {
+  display: flex;
+  justify-items: center;
+  margin: 0 auto;
+  height: 12rem;
+  justify-content: space-around;
+  font-size: 30px;
+  width: 80%;
+}
+
+.net h2 {
+  height: 10rem;
+  color: #5d5f63;
+}
+
+.naver {
+  font-weight: 1000;
+}
+
+.ggl {
+  font-weight: 100;
+}
+
+.foot-sctn2 {
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  margin: 0 auto;
+  padding: 5rem 0;
+  width: 60%;
+}
+
+.foot-sctn2 .common {
+  width: 70%;
+}
+
+.foot-sctn2 img {
+  height: 5rem;
+}
+
+.nav_bar {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  .nav_bar {
+    position: absolute;
+    background-image: url(./icon/next-bg.jpg);
+    height: 45rem;
+    width: 100%;
+  }
```

Comment:
- [ ]  Great job using media queries, however, it's one of the project requirements to build a mobile-first website, which means the styling should be first for mobile and we use media queries for other sized devices. Please fix that.